opengemini/write.go

```diff
@@ -56,3 +56,46 @@ func (c *client) WriteBatchPoints(database string, bp *BatchPoints) error {
 	}
 	return nil
 }
+
+func (c *client) WritePoint(database string, point *Point, callbackFunc func(error)) error {
+	var buffer bytes.Buffer
+
+	var writer io.Writer
+
+	if c.config.GzipEnabled {
+		writer = gzip.NewWriter(&buffer)
+	} else {
+		writer = &buffer
+	}
+
+	if _, err := io.WriteString(writer, point.String()); err != nil {
+		return err
+	}
+
+	if closer, ok := writer.(io.Closer); ok {
```

Comment:
why we must cast to `io.Closer` type?

Reply:
GzipWriter’s Close method closes itslef by flushing any unwritten data to the underlying io.Writer and writing the GZIP footer.It does not close the underlying io.Writer.